server/controllers/transferwise.ts

```diff
@@ -50,14 +50,27 @@ export async function payBatch(
     if (!remoteUser.isAdmin(host.id)) {
       throw new errors.Unauthorized('User must be admin of host collective');
     }
-    // TODO: ensure expense ids are sent with the OTT retry request
     const expenseIds = body?.expenseIds?.map(id => idDecode(id, IDENTIFIER_TYPES.EXPENSE));
     const expenses = await models.Expense.findAll({
       where: { id: { [Op.in]: expenseIds } },
-      include: [{ model: models.PayoutMethod, as: 'PayoutMethod' }],
+      include: [
+        { model: models.PayoutMethod, as: 'PayoutMethod', required: true },
+        {
+          model: models.Collective,
+          as: 'collective',
+          where: { HostCollectiveId: host.id },
```

Comment:
Though we'll probably prevent that if there are pending expenses, collectives can change hosts and past expenses should be attached to the host of the collective at submission's time (https://github.com/opencollective/opencollective/issues/4271). I would add a small TODO here:

```suggestion
          where: { HostCollectiveId: host.id }, // TODO: We should ideally use the host attached to the expense. See https://github.com/opencollective/opencollective/issues/4271
```